chia/cmds/wallet_funcs.py

```diff
@@ -543,15 +569,21 @@ async def get_wallet(wallet_client: WalletRpcClient, fingerprint: int = None) ->
                 row += f" ({current_sync_status})"
             print(row)
         val = None
+        prompt: str = (
+            f"Choose a wallet key [1-{len(fingerprints)}] ('q' to quit, or Enter to use {logged_in_fingerprint}): "
+        )
         while val is None:
-            val = input("Enter a number to pick or q to quit: ")
+            val = input(prompt)
             if val == "q":
                 return None
-            if not val.isdigit():
+            elif val == "" and logged_in_fingerprint is not None:
+                fingerprint = logged_in_fingerprint
+                break
+            elif not val.isdigit():
                 val = None
             else:
                 index = int(val) - 1
-                if index >= len(fingerprints):
+                if index < 0 or index >= len(fingerprints):
```

Comment:
This fixed an issue I found where you could enter `0` when selecting a key which would lead to `index == -1`